marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/CreateGranularPrivilegesCommand.java

```diff
@@ -44,44 +42,84 @@ public Integer getUndoSortOrder() {
     public void execute(CommandContext context) {
         final String finalDbName = hubConfig.getDbName(DatabaseKind.FINAL);
         final String stagingDbName = hubConfig.getDbName(DatabaseKind.STAGING);
-
-        // Once RMA supports pseudo functions - https://bugtrack.marklogic.com/53540 - then we won't need to get the
-        // resource IDs here and can use resource names instead.
-        ResourcesFragment databasesXml = new DatabaseManager(context.getManageClient()).getAsXml();
-        final String finalDbId = databasesXml.getIdForNameOrId(finalDbName);
-        final String stagingDbId = databasesXml.getIdForNameOrId(stagingDbName);
+        final String finalTriggersDbName = hubConfig.getDbName(DatabaseKind.FINAL_TRIGGERS);
+        final String stagingTriggersDbName = hubConfig.getDbName(DatabaseKind.STAGING_TRIGGERS);
 
         PrivilegeManager mgr = new PrivilegeManager(context.getManageClient());
         Privilege p = new Privilege(null, "admin-database-clear-" + finalDbName);
         p.setKind("execute");
-        p.setAction("http://marklogic.com/xdmp/privileges/admin/database/clear/" + finalDbId);
+        p.setAction("http://marklogic.com/xdmp/privileges/admin/database/clear/$$database-id(" + finalDbName+ ")");
```

Comment:
What just the database ID by itself not working?

Reply:
It does work, I saw this comment "Once RMA supports pseudo functions - https://bugtrack.marklogic.com/53540 - then we won't need to get the	 resource IDs here and can use resource names instead" and the bug had been shipped, so changed them to use pseudo functions

Reply:
Oh interesting - since that's the case, we could externalize this config into files, as we don't need code anymore to look up the resource IDs. But I think we can hold off on that for now.